pepsico/config-dev-sample.yaml

```diff
@@ -0,0 +1,10 @@
+# Configuration overrides for development. Copy this file to config-dev.yaml and edit as needed.
+
+mode: debug  # debug, devel or prod
+dev_server_interface: 127.0.0.1
+dev_server_port: 8050
+dev_processes: 4
+loglevel: info
```

Comment:
This doesn't appear to be used anywhere. I was tipped off by the fact that there's no default for it in config-defaults.yaml. Please remove from here and from enacts/config-dev-sample.yaml.